app/src/main/AndroidManifest.xml

```diff
@@ -10,7 +10,9 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme" >
+        android:theme="@style/AppTheme"
+        android:hardwareAccelerated="true"
```

Comment:
Is this very important, or only kinda important?

Reply:
Rendering is faster in some cases